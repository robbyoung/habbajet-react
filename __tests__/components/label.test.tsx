import React from 'react';
import renderer from 'react-test-renderer';
import Label from '../../app/components/label';

describe('Label Component', () => {
    it('will render a label with a title and content', () => {
        const component = renderer.create(
            <Label title="Title" content="Content" color="#036bfc" />,
        );
        expect(component.toJSON()).toMatchSnapshot();
    });

    it('can take a larger font size', () => {
        const component = renderer.create(
            <Label
                title="Title"
                content="Content"
                color="#036bfc"
                contentSize={70}
            />,
        );
        expect(component.toJSON()).toMatchSnapshot();
    });
});
