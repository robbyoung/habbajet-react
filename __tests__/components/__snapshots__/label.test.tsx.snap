// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label Component will render a label with a title and content 1`] = `
<View>
  <Text
    style={
      Array [
        Object {
          "fontFamily": "Abel",
          "fontSize": 20,
          "margin": 0,
          "padding": 0,
        },
        Object {
          "color": "#036bfc",
        },
      ]
    }
  >
    Title
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "Abel",
        "fontSize": 30,
        "marginTop": -10,
        "padding": 0,
      }
    }
  >
    Content
  </Text>
</View>
`;
